### PR TITLE
Cody Windeknecht & Glenn-David Daniel --- React-Testing

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   "devDependencies": {
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
-    "sinon": "^4.4.8"
+    "sinon": "^4.4.8",
+    "jest": "22.4"
```
Comment (mixelpixel) [inline package.json]: Jest is actually included already in the 'react' dependencies... as far as I know they come along when `create-react-app` is used and are part of the modules installed when 'react' is already in the package.json file.

Comment (d2rd) [reply package.json]: So am I to:
1) Delete /node_modules
2) Delete ` "jest":  "22.4" ` . from "devDependencies"
3) Leave in the new entry `"jest": { "testEnvironment": "node" }`
For this end result:

```
"devDependencies": {
    "enzyme": "^3.3.0",
    "enzyme-adapter-react-16": "^1.1.1",
    "sinon": "^4.4.8",
    "jest": "22.4"
  },
  "jest": {
    "testEnvironment": "node"
  }

```

Comment (mixelpixel) [reply package.json]: Close. You'll also want to remove
```
,
  "jest": {
    "testEnvironment": "node"
  }
```

Comment (mixelpixel) [reply package.json]: Just like it is in the Lambda School repository

**Pull request closed without merging**